packages/components/src/menu-items-choice/stories/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import MenuItemsChoice from '..';
+import MenuGroup from '../../menu-group';
+
+const meta: ComponentMeta< typeof MenuItemsChoice > = {
+	component: MenuItemsChoice,
+	title: 'Components/MenuItemsChoice',
+	argTypes: {},
+	parameters: {
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof MenuItemsChoice > = ( args ) => {
+	const [ choice, setChoice ] = useState( args.value ?? '' );
+	const [ hovered, setHovered ] = useState< string | undefined >();
+
+	return (
+		<MenuGroup label="Editor">
+			<MenuItemsChoice
+				{ ...args }
+				value={ choice }
+				onSelect={ ( value ) => setChoice( value ) }
+				onHover={ ( value ) => setHovered( value ) }
+			/>
+			<div>Currently hovered value: { hovered ? hovered : '-' }</div>
```

Comment:
For callbacks like the `onHover` here, we usually use the Storybook [Actions](https://storybook.js.org/docs/react/essentials/actions) feature so we don't have to render callback indicators like "Currently hovered...". Here's an example implementation:

https://github.com/WordPress/gutenberg/blob/5115017b4129e942cce903a334f9d0350e21a0fe/packages/components/src/search-control/stories/index.tsx#L19-L21

https://github.com/WordPress/gutenberg/blob/5115017b4129e942cce903a334f9d0350e21a0fe/packages/components/src/search-control/stories/index.tsx#L39-L42

Reply:
TIL! Thanks. Changed it via https://github.com/WordPress/gutenberg/pull/47180/commits/b46f7706f371b30742b4ccb93fac7dae8b69a605.